bundles/mapping/mapmodule/MapModuleTextButton.jsx

```diff
@@ -55,7 +55,8 @@ const StyledButton = styled(Button)`
     }
 `;
 
-const ButtonText = styled('span')`
+const ButtonText = styled('div')`
```

Comment:
Should we add a comment here that these handles cases where the text is given directly as string and the duplicated ellipsis on Message-tag is because these don't work with for example Message-tag?